cmake/onnxruntime_python.cmake

```diff
@@ -20,10 +20,6 @@ file(GLOB onnxruntime_pybind_srcs CONFIGURE_DEPENDS
   ${onnxruntime_pybind_srcs_pattern}
   )
 
-if(NOT onnxruntime_PYBIND_EXPORT_OPSCHEMA)
```

Comment:
I would probably this this for the MINIMAL BUILD. The schema information contains all the documentation and it could be removed from the memory for this particular build.

Reply:
I could be mistaken, but I would expect devs wanting to build for minimal_build would probably not want to build the pybind layer. @snnn do you have more context here?

I can change this macro so that it now uses minimal_build to control whether or not the schema is visible in the package if that is the consensus.

Reply:
I think they usually do not python.

Reply:
while we don't have minimal build python packages, it is still possible to do a minimal build with python enabled. that said, I think it's ok to leave this enabled for simplicity as the additional core code is minimal.

FYI @skottmckay

Reply:
Let's keep it that way then. Thanks for the explanations.